components/automate-chef-io/content/docs/api-tokens.md

```diff
@@ -10,124 +10,48 @@ toc = true
     weight = 70
 +++
 
-{{< info >}}
-This content is specific to IAM v1. See [IAM v2 Overview]({{< ref "iam-v2-overview.md#" >}}) for IAM v2 specific information.
-{{< /info >}}
+## Overview
+Chef Automate API Tokens allow access to the Chef Automate API.
 
-Chef Automate has two different types of API tokens: administrative and standard.
-With an administrative token you can access the entire Chef Automate API--including administrative tasks such as [managing local users]({{< relref "users.md" >}}), [managing local teams]({{< relref "teams.md" >}}), and [refining IAM permissions]({{< relref "iam-v1-overview.md" >}}).
-Standard tokens have much more limited permissions; they are designed for Chef Clients and InSpec Agents to send data to Chef Automate.
-You can also use a standard token to access any part of the Chef Automate API if you write a policy granting it specific access. However, granting a user access to the ``auth:tokens`` resource also gives that user access to the administrative token.
-
-Before you follow these instructions, we recommend you install the JSON processor [jq](https://stedolan.github.io/jq/) to ensure readable output. Without it, some commands may need to be modified.
-
-## Viewing API Tokens
-
-![API Tokens](/images/docs/admin-tab-API-tokens-list.png)
-
-From the **Settings** tab, navigate to the _API Tokens_ page. Admin users and non-admin users with admin-like authorization defined in policies, will see all of the tokens registered in Chef Automate.
-
-For more information on defining policies, see [IAM v1 policies]({{< relref "iam-v1-policies.md" >}}).
+Permission on the `iam:tokens` resource is required to interact with tokens. Any user that is part of the `admins` team, or the `Administrator` policy will have this permission, otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.
```

Comment:
```suggestion
Permission on the `iam:tokens` resource is required to interact with tokens. Any user that is part of the `admins` team, or the `Administrator` policy will have this permission. Otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.
```

Reply:
Side note: Why is `Administrator` single and `Editors/Viewers` plural? 😩